src/sentry/features/__init__.py

```diff
@@ -114,7 +114,7 @@
 default_manager.add("organizations:required-email-verification", OrganizationFeature, True)  # NOQA
 default_manager.add("organizations:rule-page", OrganizationFeature)
 default_manager.add("organizations:selection-filters-v2", OrganizationFeature, True)
-default_manager.add("organizations:session-replay", OrganizationFeature)
+default_manager.add("organizations:session-replay", OrganizationFeature, True)
```

Comment:
@Zylphrex Do we need to do anything else (besides configuring flagr)?

Reply:
No, after setting this to `True`, you just need to configure flagr